app/javascript/src/components/Dashboard/Contacts/Table.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+
+import { Table as NeetoUITable } from "neetoui";
+
+import { renderContactDetails, renderText, renderDropdown } from "./utils";
+
+const Table = ({ contacts = [], onDeleteContact }) => {
+  const [pageNumber, setPageNumber] = useState(1);
+
+  const COLUMN_DATA = [
+    {
+      title: "Name & Role",
+      dataIndex: "name_and_role",
+      key: "name_and_role",
+      width: "30%",
+      render: renderContactDetails,
+    },
+    {
+      title: "Email",
+      dataIndex: "email",
+      key: "email",
+      width: "30%",
+      render: renderText,
+    },
+    {
+      title: "Created At",
+      dataIndex: "created_at",
+      key: "created_at",
+      width: "30%",
+      render: renderText,
+    },
+    {
+      title: "",
+      dataIndex: "icon_button",
+      key: "icon_button",
+      width: "10%",
+      render: (_, { id }) => renderDropdown({ onDeleteContact, id }),
+    },
+  ];
```

Comment:
Why's this constant kept within the presentation layer? Move this to a `constants` file within the closest scope.

Refer: https://academy.bigbinary.com/learn-react/react-best-practices/structuring-code

Reply:
I initially did keep it in the constants file but the UI broke as the id of contact is being passed in line 37 `render: (_, { id }) => renderDropdown({ onDeleteContact, id }),`. I'll see if something else can be done.

Reply:
@yedhink https://youtu.be/4pnhKd41Cn0

Reply:
@yedhink I fixed it by storing the return value of the function `COLUMN_DATA` from constants file to a variable. Then, providing the variable to the columnData param in NeetoUITable.